webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/api/DataStructures.scala

```diff
@@ -246,7 +257,10 @@ object ApiAuthorization       {
   case object None                               extends ApiAuthorization { override val kind = ApiAuthorizationKind.None }
   case object RW                                 extends ApiAuthorization { override val kind = ApiAuthorizationKind.RW   }
   case object RO                                 extends ApiAuthorization { override val kind = ApiAuthorizationKind.RO   }
-  final case class ACL(acl: List[ApiAclElement]) extends ApiAuthorization { override def kind = ApiAuthorizationKind.ACL  }
+  final case class ACL(acl: List[ApiAclElement]) extends ApiAuthorization {
+    override def kind = ApiAuthorizationKind.ACL
+    override def toString: String = acl.map(_.display).mkString(";")
```

Comment:
can you add an alias "debugString" and use it where appropriate ? The jvm platform force us to have `.toString`, but it makes for unbeliable hard to correct bugs (related to string concat when it's not what you want at all) and should be avoided. So as a good practice, we don't use "toString" but `.debugString` for which the intent is clear (and we wait for a linter rule that can scream at use when toString is used). 

Plus, it makes it clear that "debugString" is not something to try to parse again, what `toString` never does, make assumption quite unclear.

Reply:
Yes that is the cleaned solution, I'll have to also change the culprit log on auth-backends and call `.debugString` on it then :see_no_evil:

Reply:
renamed the `toString` and added it also in https://github.com/Normation/rudder-plugins/pull/675 :white_check_mark: